resources/RegexGrammar.pp

```diff
@@ -122,7 +123,7 @@
 %token character_type            \\([CdDhHNRsSvVwWX]|[pP]{[^}]+})
 %token anchor                    \\([bBAZzG])|\^|\$
 %token match_point_reset         \\K
-%token literal                   \\.|.
+%token literal                   \\.|.|\n
```

Comment:
Do we need \r here, too?

Reply:
`\r` is very much worth parsing because even in `\r\n` if you can't parse `\r` then you fail, but it is included in `.` so all good.